src/main/java/ezvcard/util/JCardPrettyPrinter.java

```diff
@@ -0,0 +1,101 @@
+package ezvcard.util;

```

Comment:
Can you move this class to the `ezvcard.io.json` package?  I think it makes more sense to put it there because it has to do with serializing jCards.